slacker/session.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+class Session:
+  __instance = None
+
+  def __init__(self):
+    """Get instance of Session via Session.get()."""
+    if not Session.__instance:
```

Comment:
Do we need this check since we are handling the check within the static `get` member.

Reply:
Yes, because we are setting the default values and the value of the instance only here (we only have the `self` here and not in the static `get`). Technically I wanted the constructor to be private, like you would do normally but that is not possible unfortunately.